### DISCUSSION TITLE
ASIO devices

### DISCUSSION THREAD

**Discussion by WardinT** [File-New-Project/EarTrumpet#770] [Q&A]
Does EarTrumpet support all kinds of input/output including ASIO?

**Comment by davuxcom** [comment]
Probably not. We use [WASAPI](https://docs.microsoft.com/en-us/windows/win32/coreaudio/wasapi?redirectedfrom=MSDN) which appears to be totally bypassed for ASIO drivers.

**Comment by MatDepInfo** [comment]
Too bad, that would be great!
